### PR TITLE
Document the $inline tag

### PR DESCRIPTION
**Pull request by fmoor**
fixes https://github.com/edgedb/edgedb-go/issues/284

### CODE REVIEW

File: rstdocs/index.rst [hunk 1]
```diff
@@ -174,6 +174,21 @@ using sets as parameters.
 Nested structures are also not directly allowed but you can use `json <https://www.edgedb.com/docs/edgeql/insert#bulk-inserts>`_
 instead.
 
+By default EdgeDB will ignore embedded structs when marshaling/unmarshaling.
+To treat an embedded struct's fields as part of the parent struct's fields
+tag the embedded struct with `edgedb:$inline`.
```
Comment (raddevon) [inline rstdocs/index.rst:179]:
```suggestion
tag the embedded struct with ``edgedb:$inline``.
```

File: rstdocs/index.rst [hunk 2]
```diff
@@ -174,6 +174,21 @@ using sets as parameters.
 Nested structures are also not directly allowed but you can use `json <https://www.edgedb.com/docs/edgeql/insert#bulk-inserts>`_
 instead.
 
+By default EdgeDB will ignore embedded structs when marshaling/unmarshaling.
+To treat an embedded struct's fields as part of the parent struct's fields
```
Comment (raddevon) [inline rstdocs/index.rst:178]: Make sure I've interpreted this correctly. I wasn't 100% certain how to interpret it.
```suggestion
To treat an embedded struct's fields as part of the parent struct's fields,
```

**Review by raddevon — APPROVED** [review]
Two minor change suggestions, but it looks good otherwise!

File: rstdocs/index.rst (hunk 1 above)
Comment (fmoor) [reply rstdocs/index.rst:179]: Good catch!

The `rst` files are generated from the go documentation comments. Go documentation comments don't have anything analogous to the double back ticks in `rst`. The back ticks here are literally what the user should put in their source `` `edgedb:"$inline"` ``. I guess the generator needs to be updated to escape the back tics.

Also looks like I forgot the quotes!

**Pull request merged**